docs/sources/setup/install/helm/_index.md

```diff
@@ -22,6 +22,17 @@ This guide references the Loki Helm chart version 3.0 or greater and contains th
 
 If you are installing Grafana Enterprise Logs, follow the [GEL Helm installation](https://grafana.com/docs/enterprise-logs/<ENTERPRISE_LOGS_VERSION>/setup/helm/).
 
+## Recommended Installation
+
+The recommended installation method for initial deployments is to use the [Loki Simple Scalable Helm chart]({{< relref "./install-scalable" >}}). This chart provides a simple scalable deployment mode for Loki, separating execution paths into read, write, and backend targets. For small to medium-sized deployments, this chart is a good starting point.
```

Comment:
great specific opinionated starting point. I'm not wondering what you think I should do.